src/sagemaker/config/config_schema.py

```diff
@@ -645,6 +650,24 @@ def _simple_path(*args: str):
                                         },
                                     },
                                 },
+                                ESTIMATOR: {
+                                    TYPE: OBJECT,
+                                    ADDITIONAL_PROPERTIES: False,
+                                    PROPERTIES: {
+                                        DEBUG_HOOK_CONFIG: {
+                                            TYPE: "boolean",
+                                            "description": (
+                                                "Sets a boolean for `debugger_hook_config` of"
+                                                "Estimator which will be then used for training job"
+                                                "API call. As of today, config_schema does not"
+                                                "support the dict as a valid value to be provided."
+                                                "In case, we decide to support it in the future,"
+                                                "we can add a new schema for it under"
+                                                "`SageMaker.TrainingJob` config path."
```

Comment:
How about something slightly stronger:

>Today, the config_schema does not"
"support a dictionary as a valid value to be provided."
"In the future to add support for DebugHookConfig as a dictionary,"
"that schema should be added under the config path"
"`SageMaker.TrainingJob` instead of here."

and consider explaining "why"

Reply:
updated comment